src/components/admin/form-components/withValidate.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+import { getDisplayName } from 'lib/higher-order-helpers';
+
+const withValidate = (...validators) => (WrappedField) => {
+  const ValidatedField = (originalProps) => {
+    const failedValidator = validators
```

Comment:
You can probably also name the failedValidator something better, for example simply `errorText`, no?

Reply:
I kind of find it strange to have `errorText={errorText(originalProps.value)}` later down the line. Do you prefer that though? I'm not tied to the naming here. - TODO after response.

Reply:
Wait I misread the code, it's perfect. FailedValidator  is a function, then it's perfect. Thought it was the text

Reply:
@emilgoldsmith realized I forgot to tag you in this. Do you like the semantics of `errorText={errorText(originalProps.value)`?

Reply:
Oh didn't you see the above? I like this part as it is, I just misread the code, with the js doc things will also become clearer @Helw150